From 6c19a8db2950c2fa28169d3682024ccba69581cc Mon Sep 17 00:00:00 2001
From: Haris Okanovic <haris.okanovic@ni.com>
Date: Thu, 2 Apr 2015 13:15:15 -0500
Subject: [PATCH] crypto: Remove unsupported headers in cryptlib.c on Phar Lap
 ETS

Remove unsupported headers tchar.h and signal.h in cryptlib.c
on Phar Lap ETS.

Upstream-Status: Inappropriate [Phar Lap ETS support]

---
 crypto/cryptlib.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/cryptlib.c b/crypto/cryptlib.c
index 8c9e434..f045e60 100644
--- a/crypto/cryptlib.c
+++ b/crypto/cryptlib.c
@@ -794,7 +794,7 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved)
 }
 #endif
 
-#if defined(_WIN32) && !defined(__CYGWIN__)
+#if defined(_WIN32) && !defined(__CYGWIN__) && !defined(PHARLAP)
 # include <tchar.h>
 # include <signal.h>
 # ifdef __WATCOMC__
